Tests/LibCrypto/TestBigInteger.cpp

```diff
@@ -687,6 +687,14 @@ EXPECT_EQUAL_TO(zero, -0.0);
     EXPECT_GREATER_THAN(one, -1.000001);
 }
 
+{
```

Comment:
I think this commit/test can be merged with the next commit

Reply:
Sure, I will.